Xamarin.Forms.Platform.Android/GestureManager.cs

```diff
@@ -108,7 +109,13 @@ bool DetectorsValid()
 		GestureDetector InitializeTapAndPanDetector()
 		{
 			var context = Control.Context;
-			var listener = new InnerGestureListener(new TapGestureHandler(() => View),
+			var listener = new InnerGestureListener(new TapGestureHandler(() => View, () =>
+			{
+				if (Element is Label label)
+					return label.FormattedText?.Spans;
+
+				return new List<Span>();
+			}),
```

Comment:
This is a function to return the current list of spans, when it needs to be run in the TapGestureHandler.cs